dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -198,6 +198,10 @@ const avatarHeadlineWrapper = css`
 	justify-content: space-between;
 `;
 
+const minHeightWithAvatar = css`
+	min-height: 259px;
```

Comment:
We may want to make this `rem` so that it scales with font size on apps?

Reply:
I had the same thought, but opted to leave it as `px` as it matches the existing style in `CommentLayout` and almost everything in these two layouts is set in `px` units. That said it does feel a bit brittle as it's relying on the size of the text. I'll do some testing—perhaps we could replace this value it with a `calc` to accommodate the relative size of the text and the fixed `px` size of the avatar. (That would also apply to the existing `CommentLayout`.)